### ISSUE TITLE
methods should follow checks for password

### ISSUE THREAD

**Issue by Monal5031** [systers/vms#750]
## Description
As a developer,
I need to make sure that my checks are being followed correctly,
so that I can find any inconsistencies in the code where the checks are not being executed.
Currently `create_admin()` and `create_volunteer()` create an admin and volunteer with password on which checks are not being executed this will be resolved until clean methods are defined in the models but then also we need to make sure we are using valid passwords.

## Mocks
N/A

### Update [Required]
- [ ] Update methods to use valid passwords and follow checks
- [ ] Update tests to follow these checks

## Definition of Done
- [ ] All of the required items are completed.
- [ ] Approval by 2 mentor.

## Estimation
40mins - 1 hr

**Comment by CleverFool77** [comment]
I would like to work this issue 🙂

**Comment by khatryshikha** [comment]
This issue seems to be interesting, may I take this issue?

**Comment by Monal5031** [comment]
Yeah sure go ahead :+1: make sure to setup the project successfully first

**Comment by khatryshikha** [comment]
This seems to be interesting, may I take up this issue?

**Comment by khatryshikha** [comment]
Yes, sure!

**Comment by sanjana11147** [comment]
Hii @khatryshikha are you working on this issue?
I would like to take it up.

**Comment by sanjana11147** [comment]
I don't think @khatryshikha is working on this issue. Can you assign it to me? @Monal5031

**Comment by Kishan-Dhakan** [comment]
@Monal5031 I would like to take this up next, shall give for it? :)

**Comment by Monal5031** [comment]
Please complete other issues first @Kishan-Dhakan :) It's better to have one issue assigned at a time.

**Comment by AnweshaSen** [comment]
Is this issue available to be taken? I would like to work on this then !!

**Comment by Kishan-Dhakan** [comment]
> Please complete other issues first @Kishan-Dhakan :) It's better to have one issue assigned at a time.

I understand :)

**Comment by Aimaanhasan** [comment]
Hello, I'd like to work on this issue, Can I fix it?

**Comment by abha224** [comment]
@Aimaanhasan are you still working on this issue?

**Comment by abha224** [comment]
@Aimaanhasan Pls let us know if you are still working on the issue so that we can open it up for others, in case you aren't.

**Comment by aastha985** [comment]
@satya7289 I would like to work on this issue as a part of OSH. Can you assign it to me? thanks!

**Comment by NenadPantelic** [comment]
@Monal5031 @abha224  Hi, can you please elaborate on which constraints you want to be added here. At this point, there are active constraints:
1. password must contain at least one digit, lowercase letter, an uppercase letter and one special char
2. must be at least 6 chars long
What do you suggest to add more? (I'm willing to do that)
